Pipfile

```diff
@@ -41,3 +41,9 @@ test = "pytest -vvs"
 
 [pipenv]
 allow_prereleases = true
+
+[resolver]
```

Comment:
This is going to be optional for usrers?

Reply:
Yup -- the code still gets the default platform dictionary, this just provide a mechanism to override.